util/jsonrpc-types/src/blockchain.rs

```diff
@@ -391,6 +391,13 @@ impl TxStatus {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, PartialEq, Eq, Hash, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum OutputsValidator {
```

Comment:
Please move this enum to `util/jsonrpc-types/src/pool.rs`

Reply:
Oh, I'm sorry! I forgot this. I will done it in 10 minutes!